### PR TITLE
Clean up the lxc building in travis

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
I want to clean up the things that I left pending on the travis file. This is going to be a slow task, so please ignore for now.

**Comment by sergiusens** [comment]
Bug in commit? Or just an experiment?

**Comment by elopio** [comment]
Started as an experiment, mutated to reduce the deps. Let me report a bug.

**Comment by sergiusens** [comment]
rebase!

File: .travis.yml [hunk 1]
```diff
@@ -22,20 +22,22 @@ install:
...
   # Allow the container user to do everything on the repository.
-  - sudo chmod -R 777 $(pwd)
+  - sudo chmod 777 $(pwd)
   # Mount the repository directory in the container.
   - sudo lxc config device add xenial /dev/sda1 disk source=$(pwd) path=$(pwd)
   # Install the snapcraft dependencies.
   - sudo lxc exec xenial -- apt-get update
-  - sudo lxc exec xenial -- apt-get install -y lxd pyflakes python-flake8 python3.5 python3-apt python3-docopt python3-coverage python3-fixtures python3-flake8 python3-jsonschema python3-mccabe python3-mock python3-pep8 python3-pexpect python3-pip python3-requests python3-requests-oauthlib python3-responses python3-ssoclient python3-testscenarios python3-testtools python3-xdg python3-yaml python3-lxml squashfs-tools python3-progressbar python3-requests-toolbelt python3-petname
+  - sudo lxc exec xenial -- apt-get install -y devscripts equivs
+  - sudo lxc exec xenial -- mk-build-deps --install --tool "apt-get -y" --build-dep $(pwd)/debian/control
```
Comment (kyrofa) [inline .travis.yml]: Ah, slick!

**Comment by kyrofa** [comment]
Yeah looks good to me :+1:

**Pull request merged**